### PR TITLE
Add beta label on usage

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

This will add a BETA label on the usage page with a link to send feedback.

We need a public feedback issue to link to. 💡 

Opened this after we briefly discussed this with @laushinka in a recent social call. 🍷 

## How to test
Go to usage and notice the new beta notice on the left sidebar.

Screenshot 2022-08-25 at 11 59 52 PM

## Release Notes

```release-note
Add beta notice and label on usage
```

## Werft options:

- [x] /werft with-preview with-payment

### CODE REVIEW

File: components/dashboard/src/teams/TeamUsage.tsx
Comment (gtsiolis) [inline components/dashboard/src/teams/TeamUsage.tsx:200]: **issue(blocking)**: Before merging this, make sure there's a public feedback issue that we link to. 💡

**Comment by gtsiolis** [comment]
@jldec [WDYT](https://en.wiktionary.org/wiki/WDYT)?

**Comment by filiptronicek** [comment]
👀 I think like the label in the tab even more. It makes it clear this feature is in beta right from the get go - even before clicking on the section.

I am unsure though if this is worth giving up the nice <kbd>Send feedback</kbd> link.

**Comment by laushinka** [comment]
This looks good! For my missing context, what was the reason that it was closed before?

**Comment by gtsiolis** [comment]
@laushinka If you are referring to the closing in https://github.com/gitpod-io/gitpod/pull/12412#event-7323327424, that was by accident while rebasing and pushing no commits to the branch. 😇 

Resolved conflicts and added the missing issue link in the `<a>` element.

I've also replaced the Beta label with an Early Access label to be aligned with the upcoming changes in https://github.com/gitpod-io/website/pull/2715. Cc @jldec

**Comment by gtsiolis** [comment]
> Btw the PR didn't create any previews. Do the labels in the description work that way? I usually always create the two tags in two lines.

@laushinka If you mean the Werft options in the PR  description, they don't work inline. The PR was opened before I learned we need to use two lines for this to work, see [relevant discussion](https://gitpod.slack.com/archives/C01KGM9EBD4/p1661511736763959?thread_ts=1661508155.015629&cid=C01KGM9EBD4).

Let me update the PR description and re-trigger a preview env.

/werft with-preview with-payment

**Review by easyCZ — APPROVED** [review]
LGTM. Adding hold in case you wanted to do any additional validation or changes.
Screenshot 2022-09-14 at 11 02 45

/hold

**Comment by gtsiolis** [comment]
Thanks @laushinka and @easyCZ for taking a look, and thanks @easyCZ for adding the hold!

To unblock this, I've replaced the _Early Access_ label with a _Beta_ label so that the _Learn more_ link makes sense as _Early Access_ is a new term that does not exist _yet_ in our docs until we merge https://github.com/gitpod-io/website/pull/2715. Once https://github.com/gitpod-io/website/pull/2715 is merged we can open a small PR to rename the label. 🛹 

/unhold

**Pull request closed without merging**
